doc/whats_new.rst

```diff
@@ -295,6 +295,10 @@ Enhancements
 Bug fixes
 .........
 
+    - :func: `model_selection.tests._search._check_param_grid` now works correctly with all types
+      that extended from sequence (except string), including range (Python 3.x) and xrange
```

Comment:
Should be `Sequence` really (and "extended from" isn't precise, because any class can declare itself as implementing `Sequence` without extending anything)